src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -719,4 +675,13 @@ namespace NServiceBus
         public NServiceBus.AzureServiceBusValidationSettings UseSubscriptionPathMaximumLength(int subscriptionPathMaximumLength) { }
         public NServiceBus.AzureServiceBusValidationSettings UseTopicPathMaximumLength(int topicPathMaximumLength) { }
     }
+    public class SizeInMegabytes
+    {
+        public static NServiceBus.SizeInMegabytes Size1024;
```

Comment:
Strange, it is [here](https://github.com/Particular/NServiceBus.AzureServiceBus/blob/9628815e5ea83abf6e3a7ea89aa3126ffad49157/src/Transport/Config/ExtensionPoints/SizeInMegabytes.cs#L10)